README.md

```diff
@@ -193,7 +193,8 @@ This will delete all resources in the created namespaces and if the kubernetes c
 Please note that cleaning up the resources will take some time.
 
 ## CONTRIBUTION
- TBD
+
+Find the contributing guidelines [here](./CONTRIBUTING.md)
```

Comment:
I just changed this part to a relative path and fixed the fromating

Reply:
LGTM